src/frontend/apps/lti_site/apps/deposit/components/Dashboard/index.tsx

```diff
@@ -77,7 +77,6 @@ const Dashboard = () => {
 
   let content: JSX.Element = <Fragment />;
   switch (useFileDepositoryStatus) {
-    case 'idle':
```

Comment:
idle does not exist anymore ? Is it replace by something else ?

Reply:
https://tanstack.com/query/v4/docs/react/guides/migrating-to-react-query-4#the-idle-state-has-been-removed

Reply:
But yes you're right, it is replaced actually.
[fixup! 🚨(frontend) react-query fix left errors](https://github.com/openfun/marsha/pull/2340/commits/d40ddca632dbd6a6c1edd3133c0dc0911ef464f8)